Tutorials/CNTK_209A_Berkeley_Segmentation_Dataset_loader.ipynb

```diff
@@ -0,0 +1,367 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false,
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "# CNTK 209 Part A: Berkeley Segmentation Dataset loader\n",
+    "\n",
+    "## Preparing the training set\n",
+    "\n",
+    "<p>This notebook serves as data preparation for the CNTK 209 tutorial. The aim of this tutorial is to showcase how we can use tools from CNTK to train models which will perform image super-resolution. The goal of the <b>single image super-resolution (SISR)</b> problem is to upscale a given image by some factor while keeping as much image details as possible and not making the image blurry. This problem is described in more detail in the remaining notebooks of this tutorial.</p>\n",
+    "<p>It is recommended for the user to complete the tutorial CNTK 206 before this. Some basic familiarity with deep convolutional networks is also welcome.</p>\n",
+    "<p>We will be using <b>Berkeley Segmetation Dataset (BSDS)</b>. It contains 300 images which we will download and then prepare for training of the models we will use. Image dimensions are 481 x 321 and 321 x 481.</p>"
```

Comment:
We should also mention the environment we used (i.e. Python 3.5, CNTK 2.x, etc.)